apps/wing/src/cli.ts

```diff
@@ -64,6 +64,12 @@ async function main() {
       "Compiles a Wing program and runs all functions with the word 'test' or start with 'test:' in their resource identifiers"
     )
     .argument("<entrypoint...>", "all entrypoints to test")
+    .addOption(
+      new Option("-t, --target <target>", "Target platform")
+        .choices(["tf-aws", "sim"])
+        .makeOptionMandatory()
```

Comment:
I think defaulting to sim still makes sense here.

Reply:
Me too, and while we are on it. Let's also make sim the default for wing compile

Reply:
+1 feels like it makes sense to make sim "first amongst equals"

Reply:
Updated to make sim the default for `compile` and `test` commands